src/mbtest/imposters/stubs.py

```diff
@@ -21,11 +21,15 @@ def __init__(
         responses: Optional[Union[BaseResponse, Iterable[BaseResponse]]] = None,
     ) -> None:
         if predicates:
-            self.predicates = predicates if isinstance(predicates, Sequence) else [predicates]
+            self.predicates = (
```

Comment:
Are you running `black` with different settings than I am, I wonder? This shouldn't need reformatting.

The command to run should be:

    black -l100 -tpy36 src/ tests/ setup.py

I run this via `tox` myself.